drivers/bluetooth/hci/rpmsg.c

```diff
@@ -198,6 +198,9 @@ static const struct bt_hci_driver drv = {
 	.open		= bt_rpmsg_open,
 	.send		= bt_rpmsg_send,
 	.bus		= BT_HCI_DRIVER_BUS_IPM,
+#if defined(CONFIG_BT_LL_SW_SPLIT)
```

Comment:
wait a second, this Kconfig will not be set in the application core :-(

@carlescufi comments?

Reply:
Something similar was done in https://github.com/zephyrproject-rtos/zephyr/commit/b0c294ce48744d31d0ecb1c6da7519ba90b19786. If the controller behaves somewhat different than the host would expect, the host needs to be aware of this. Maybe we can add those quirks to Kconfig?